### PR TITLE
Update animationHelpers.js

### PR DESCRIPTION
**Pull request by tech-extremists**

### CODE REVIEW

**Comment by tech-extremists** [comment]
!cond_accept

**Review by atakanserifoglu — COMMENTED** [review]
!cond_accept

**Comment by atakanserifoglu** [comment]
!cond_accept

File: animationHelpers.js [hunk 1]
```diff
@@ -1,19 +1,11 @@
...
- * @id 22102100
- * 
- * @author Ahmet Kaan Sever
- * @id 22102278
- */
-
+vertexArray = [];
 // Saves the current theta and translation values of the current keyframe.
 function saveCurrKeyframe(){
     saveKeyframeButton.onclick = function () {
```
Comment (atakanserifoglu) [inline animationHelpers.js:4]: sdsd

**Pull request closed without merging**